x-pack/plugins/observability_ai_assistant/public/analytics/schema.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RootSchema } from '@kbn/analytics-client';
+import { Message } from '../../common';
+import type { Feedback } from '../components/feedback_buttons';
+
+export const MESSAGE_FEEDBACK = 'message_feedback';
+
+export interface MessageFeedback extends Message {
+  feedback: Feedback;
+}
+
+export interface TelemetryEvent {
+  eventType: typeof MESSAGE_FEEDBACK;
```

Comment:
Do you need to put at `as const` on `MESSAGE_FEEDBACK` to narrow it to the string literal?